### PR TITLE
Fixes #2346

### LINKED ISSUE #2346: New feature request:  Global command
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):  Feature Request

**What happened**: 
Upon typing ":g/vpcid/ d" and hitting Enter/Return key, shows the message "E492: Not an editor command. g/vpcid/ d"

**What did you expect to happen**:
I expect all the lines that match the pattern specified between the slashes, to be deleted.  I would like the same functionality as the global command g and v (refer [http://vim.wikia.com/wiki/VimTip227](Power of g)).  This feature is especially useful when dealing with data files.

**How to reproduce it (as minimally and precisely as possible)**:
Enter the above command for a file with some text pattern that is present in the file.  And hit Enter/Return key.

**Environment**:

- Extension (VsCodeVim) version: 0.10.13
- VSCode version: 1.19.3
- OS version: macOS Sierra (v10.12.6)

### PR DESCRIPTION
**Pull request by Chillee**
https://github.com/VSCodeVim/Vim/issues/2346#issuecomment-361826197

explains the problem and the fix.

### CODE REVIEW

File: src/cmd_line/commandLine.ts [hunk 1]
```diff
@@ -38,14 +38,17 @@ export class CommandLine {
...
-      console.log(e);
       if (e instanceof VimError) {
-        StatusBar.SetText(
-          `${e.toString()}. ${command}`,
-          vimState.currentMode,
-          vimState.isRecordingMacro,
-          true
-        );
+        if (e.code === ErrorCode.E492) {
+          await Neovim.command(vimState, command);
```
Comment (jpoon) [inline src/cmd_line/commandLine.ts]: Even for users that haven't configured to use neovim, you want to use neovim as a fallback

Comment (Chillee) [reply src/cmd_line/commandLine.ts]: My bad. You're right, there should be another check here.

**Comment by Chillee** [comment]
Finally, it works. I didn't realize Travis built with some version of the main branch merged with the PR.

**Comment by jpoon** [comment]
>  I didn't realize Travis built with some version of the main branch merged with the PR.

@Chillee what do you mean? Travis will build this particular PR.

**Pull request merged**